build/index.asset.php

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```

Comment:
@roshan-singh-qa Is this PR still valid? I'm seeing some merge conflicts that need to be resolved.